src/promdens/promdens.py

```diff
@@ -609,13 +611,17 @@ def plot_spectrum(ics: InitialConditions) -> None:
     axs[1].minorticks_on()
     axs[1].tick_params('both', direction='in', which='both', top=True, right=True)
 
-    axs[2].plot(ics.spectrum[0]/ics.evtoau, ics.spectrum[-1], color=colors[-1], label='Total spectrum')
-    axs[2].fill_between(ics.spectrum[0]/ics.evtoau, ics.spectrum[-1]*0, ics.spectrum[-1], color=colors[-1], alpha=0.2)
+    energy_ev = ics.spectrum[0]/ics.evtoau
```

Comment:
Not really sure about the names of these variables

Reply:
I kept `energy_ev` for now. I am not quite satisfied with either `spectrum_energy_ev` or for example `energy_grid_ev`, as they are both longer and not really consistent with other naming in these functions (since we now have e.g. `t_fs`).